main.ts

```diff
@@ -20,27 +20,30 @@ import options from "./options.ts";
 
 import { routes as completionsV2Routes } from "./completions_v2.ts";
 
-const docland = "https://doc.deno.land/";
 await setup({
   resolveHref(current, symbol) {
     // FIXME(bartlomieju): special casing for std here is not ideal
     if (symbol && current.startsWith("/std")) {
       current = `https://deno.land${current}`;
     }
-    return symbol ? `${docland}${current}/~/${symbol}` : current;
+    return symbol ? `${current}?s=${symbol}` : current;
   },
   lookupHref(
-    current: string,
-    namespace: string | undefined,
-    symbol: string,
+    _current: string,
+    _namespace: string | undefined,
+    _symbol: string,
   ): string | undefined {
+    return undefined;
     // FIXME(bartlomieju): special casing for std here is not ideal
-    if (current.startsWith("/std")) {
+    /*if (current.startsWith("/std")) {
```

Comment:
Do we really want to leave commented out code in here?

Reply:
i guess it is not necessary